guides/v2.3/config-guide/elasticsearch/es-config-stopwords.md

```diff
@@ -126,3 +127,21 @@ To change the directory:
     ```
 
 Save your changes to `di.xml` and exit the text editor.
+
+## To change the directory from your module {#to-change-directory-stopwords}
+
+1. [Create a new module](https://devdocs.magento.com/guides/v2.3/extension-dev-guide/build/module-file-structure.html){:target="_blank"}
+1. In your module `etc/di.xml` add instructions:
+
+    ```xml
+    <type name="Magento\Elasticsearch\SearchAdapter\Query\Preprocessor\Stopwords">
+        <arguments>
+           <argument name="stopwordsModule" xsi:type="string">Your_Module</argument>
+           <argument name="stopwordsDirectory" xsi:type="string">stopwords</argument>
+        </arguments>
+    </type>
+    ```
+
+1. In your module, create the directory `etc/stopwords`, with the corresponding `.csv`.
+[files inside.](https://github.com/magento/magento2/tree/2.3-develop/app/code/Magento/Elasticsearch/etc/stopwords){:target="_blank"}
```

Comment:
Please, use the relative blob links instead of the direct absolute links. You can find an example at the very bottom of the [following](https://raw.githubusercontent.com/magento/devdocs/2946d41c63a2747ddf87adf747512e8742c58cca/guides/v2.2/javascript-dev-guide/widgets/widget_accordion.md) file.

I.e. 

```javascript
[lib/web/mage/accordion.js]: {{ site.mage2bloburl }}/{{ page.guide_version }}/lib/web/mage/accordion.js
```

Reply:
Hi @rogyar the changes were taken care of but i had to add new variable